src/build/arg/mod.rs

```diff
@@ -54,18 +58,13 @@ type Id = u64;
 #[allow(missing_debug_implementations)]
 #[derive(Default, Clone)]
 pub struct Arg<'help> {
+    pub(crate) id: Id,
+    pub(crate) name: &'help str,
+    pub(crate) help: Option<&'help str>,
+    pub(crate) long_help: Option<&'help str>,
+    pub(crate) blacklist: Option<Vec<Id>>,
```

Comment:
What's up with these? Why do the other fields still contain hidden?

Reply:
Because WIP 😓